### ISSUE TITLE
biblioportal update issues (minor)

### ISSUE THREAD

**Issue by graybeal** [ncbo/bioportal-project#70]
With the 2.5 release, Biblioportal lost its custom header, and the API isn't working any more.

Michelle says neither is a rush, and of course they aren't paying us anyway. (Though I will talk to Phillip S next week about that...)  So we can wait until we have a fast and happy path on these.

**Comment by michelleif** [comment]
also fyi (not asking you to fix yet just letting you know), the Properties tab in Biblioportal is not loading anything, where the same ontology in BioPortal does. Example https://bioportal.bioontology.org/ontologies/BIBLIOTEK-O/?p=properties
vs
http://biblio.ontoportal.org/ontologies/BIBLIOTEK-O/?p=properties

**Comment by jvendetti** [comment]
@alexskr - not certain, but I wonder if there could be a configuration issue? For the properties tab that Michelle references above, I see the following CORS error in the browser developer console:

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://171.67.213.105/ajax/properties/tree?apikey=&ontology=BIBLIOTEK-O&no_context=true. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).
```

I also see this error on every page:

```
Loading failed for the <script> with source “http://171.67.213.105/javascripts/bp_crossdomain_autocomplete.js?_=1520900809924”.
```

**Comment by graybeal** [comment]
Could explain API issue also.

**Comment by alexskr** [comment]
yep,  it was a configuration issue.

**Comment by jvendetti** [comment]
Properties now appear in the UI:

![screenshot 2018-03-13 11 31 31](https://user-images.githubusercontent.com/1696923/37362342-264bc8e4-26b2-11e8-9681-c19b46469cc2.png)

@alexskr - not sure if the other problem reports in the initial comment were addressed? I looked at the BibliPortal site and I see the custom logo, so I'm not sure what the problem is there?

Also, I don't know the URL of their REST endpoint. I tried http://data.biblioportal.org/ and http://data.biblioportal.ontoportal.org/, but neither work.
